h2o-algos/src/main/java/hex/tree/PlattScalingHelper.java

```diff
@@ -0,0 +1,124 @@
+package hex.tree;
+
+import hex.Model;
+import hex.ModelBuilder;
+import hex.ModelCategory;
+import hex.glm.GLM;
+import hex.glm.GLMModel;
+import water.*;
+import water.fvec.Frame;
+import water.fvec.Vec;
+
+import static hex.ModelCategory.Binomial;
+
+public class PlattScalingHelper {
+    
+    public interface ModelBuilderWithCalibration<M extends Model<M , P, O>, P extends Model.Parameters, O extends Model.Output> {
+        void warn(String field, String msg);
+        void error(String field, String msg);
+        int nclasses();
+        Frame init_adaptFrameToTrain(Frame f, String label, String param, boolean expensive);
+        Frame getCalibrationFrame();
+        void setCalibrationFrame(Frame f);
+        void setCalibrationOutput(GLM calibrationOutput, M model);
+    }
+
+    public interface ParamsWithCalibration {
+        boolean isCVModel();
+        String weightsColumn();
```

Comment:
unfortunately yes, since ParamsWithCalibration does not extend Parameters so if I want to use an interface the only other option would be to cast ParamsWithCalibration to Parameters and access the public _fields, I went with the interface options, looks cleaner to me